modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java

```diff
@@ -36,6 +36,10 @@
     @ApiModelProperty(value = "Admin flag.")
     private boolean admin;
 
+    /** Switch user used flag. */
+    @ApiModelProperty(value = "Switch user used flag.")
+    private boolean becomeUsed;
```

Comment:
Why we need this flag?

Reply:
This flag used in frontend part for change UI